index.js

```diff
@@ -99,6 +107,7 @@ export default class QRCodeScanner extends Component {
       fadeInOpacity: new Animated.Value(0),
       isAuthorized: false,
       isAuthorizationChecked: false,
+      flashMode: props.flashMode,
```

Comment:
do we need this here if we're not modifying the flashMode internally as a state?